tutorial/documents.md

```diff
@@ -4,22 +4,21 @@ JupyterLab can be extended in two ways via:
 
 - application plugins (top level): Application plugins extend the
   functionality of JupyterLab itself, and we cover them in the
-  "Plugins" tutorial focuses.
+  [Plugins](./plugins) tutorial focuses.
 - **document widget extensions (lower level):** Document widget extensions extend
   the functionality of document widgets added to the application, and we cover
   them in this section.
 
 For this section, the term, 'document', refers to any visual thing that is
 backed by a file stored on disk (i.e. uses Contents API).
 
-The [Document Manager](http://jupyter.org/jupyterlab/classes/_docmanager_manager_.documentmanager.html)
-uses the Document Registry to create models and widgets for documents.
-
-The [File Browser](http://jupyter.org/jupyterlab/classes/_filebrowser_browser_.filebrowserwidget.html)
-uses the Document Manager when creating and opening files.
-
 The [Document Registry](http://jupyter.org/jupyterlab/classes/_docregistry_registry_.documentregistry.html)
-is the default mechanism for interacting with files in JupyterLab.
+is where document types and factories are registered.  Plugins can require
```

Comment:
@blink1073 @ian-r-rose If you can think of wording, it would be great to find a end-user friendly way to explain the registry without repeating "registered".  If nothing comes into mind immediately, just leave as is :sunny: